test/integration/apiserver/openapiv3_test.go

```diff
@@ -47,6 +47,7 @@ func TestOpenAPIV3SpecRoundTrip(t *testing.T) {
 	defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, genericfeatures.OpenAPIV3, true)()
 	controlPlaneConfig := framework.NewIntegrationTestControlPlaneConfigWithOptions(&framework.ControlPlaneConfigOptions{})
 	controlPlaneConfig.GenericConfig.OpenAPIConfig = framework.DefaultOpenAPIConfig()
+	controlPlaneConfig.GenericConfig.OpenAPIV3Config = framework.DefaultOpenAPIV3Config()
```

Comment:
nit: To prevent us from forgetting to set the OpenAPIV3Config in future tests, would it make sense to create a utility function to set up both the OpenAPI v2 and v3 configs?

Reply:
I don't think we would gain much from that since both just use the default config. Or do you mean to set both open api configs from within `NewIntegrationTestControlPlaneConfigWithOptions`?